.github/workflows/closed-issue-message.yml

```diff
@@ -1,7 +1,7 @@
 name: Closed Issue Message
 on:
-    issues:
-       types: [closed]
+  issues:
+    types: [closed]
```

Comment:
dont think you intended to bring this change into this PR - leftover merge artifact?

Reply:
ah yes thanks for the check